README.md

```diff
@@ -81,6 +81,13 @@ the only thing on the line:
 
           # A dictionary of fields to annotate on each event.
           "fields": { "type": "syslog" }
+
+          # how long to keep following a file if it's unchanged
+          # It will refollow the file if it changes but it closes 
+          # file handle. 
+          # example values: "300ms", "-1.5h" or "2h45m"
+          # valid units: "ns", "us" (or "µs"), "ms", "s", "m", "h".
```

Comment:
I like the idea here, though I think subsecond units should be removed given it is probably not appropriate to have a file with a dead time of 4ns.

I also propose a revised documentation to make it read more clearly (at least to me)

> To save on resources (open files), logstash-forwarder will close file handles
> on files that have not emitted new events after some time period.
> This setting controls that time period which we call 'dead time'.
> The file will still be monitored for new events, but just not as an
> open file.
>
> Valid units include "s" (seconds), "m" (minutes), and "h" (hours). 
> For example, "30m" is 30 minutes.

Thoughts?